recipe/build.sh

```diff
@@ -0,0 +1,8 @@
+# Copy over the include and lib dirs
+mkdir ${PREFIX}/include && cp -r ${SRC_DIR}/include/* ${PREFIX}/include

```

Comment:
I'm not sure if it is worth doing this here, but in unix it is usually better to copy files using the `install` command, that allows you to enforce correct permissions to the files, instead of copying with `cp`. I believe that if the files have correct permissions in the final package we can leave it as is.

Reply:
I did it with install